src/type/preload.ts

```diff
@@ -508,92 +508,101 @@ export type ConfirmedTips = {
   notifyOnGenerate: boolean; // 音声書き出し時の通知
 };
 
-export const configSchema = z.object({
-  inheritAudioInfo: z.boolean().default(true),
-  activePointScrollMode: z.enum(["CONTINUOUSLY", "PAGE", "OFF"]).default("OFF"),
-  savingSetting: z
-    .object({
-      fileEncoding: z.enum(["UTF-8", "Shift_JIS"]).default("UTF-8"),
-      fileNamePattern: z.string().default(""),
-      fixedExportEnabled: z.boolean().default(false),
-      avoidOverwrite: z.boolean().default(false),
-      fixedExportDir: z.string().default(""),
-      exportLab: z.boolean().default(false),
-      exportText: z.boolean().default(false),
-      outputStereo: z.boolean().default(false),
-      audioOutputDevice: z.string().default(""),
-    })
-    .default({}),
-  hotkeySettings: hotkeySettingSchema.array().default(defaultHotkeySettings),
-  toolbarSetting: toolbarSettingSchema
-    .array()
-    .default(defaultToolbarButtonSetting),
-  engineSettings: z.record(engineIdSchema, engineSettingSchema).default({}),
-  userCharacterOrder: speakerIdSchema.array().default([]),
-  defaultStyleIds: z
-    .object({
-      engineId: engineIdSchema
-        .or(z.literal(EngineId("00000000-0000-0000-0000-000000000000")))
-        .default(EngineId("00000000-0000-0000-0000-000000000000")),
-      speakerUuid: speakerIdSchema,
-      defaultStyleId: styleIdSchema,
-    })
-    .array()
-    .default([]),
-  presets: z
-    .object({
-      items: z
-        .record(
-          presetKeySchema,
-          z.object({
-            name: z.string(),
-            speedScale: z.number(),
-            pitchScale: z.number(),
-            intonationScale: z.number(),
-            volumeScale: z.number(),
-            prePhonemeLength: z.number(),
-            postPhonemeLength: z.number(),
-            morphingInfo: z
-              .object({
-                rate: z.number(),
-                targetEngineId: engineIdSchema,
-                targetSpeakerId: speakerIdSchema,
-                targetStyleId: styleIdSchema,
-              })
-              .optional(),
-          })
-        )
-        .default({}),
-      keys: presetKeySchema.array().default([]),
-    })
-    .default({}),
-  defaultPresetKeys: z.record(voiceIdSchema, presetKeySchema).default({}),
-  currentTheme: z.string().default("Default"),
+// ルート直下にある雑多な設定値
+export const RootMiscSettingSchema = z.object({
```

Comment:
```suggestion
export const rootMiscSettingSchema = z.object({
```
schemaはcamelCaseのはず？

Reply:
確かに！　一括変換します！